platform-grpc-service-framework/src/main/java/org/hypertrace/core/serviceframework/grpc/GrpcPlatformServiceContainer.java

```diff
@@ -199,6 +199,23 @@ public boolean healthCheck() {
     }
   }
 
+  private Duration[] generateLatencyHistogramBuckets() {
+    ArrayList<Duration> histogram_buckets = new ArrayList<>();
+    int factor = 3;
```

Comment:
I thought we had discussed 5 as the factor - any reason switched to 3? No strong preference. Maybe a bit towards the larger buckets (larger factor) since they save a few metrics and are better aligned with how we typically group data. Either way I'd at a minimum pull it into a constant.

Reply:
With the default max set to 10000ms and if factor is set to 5, we get very sparse buckets - [10000, 2000, 400, 80, 16] (especially in the range of 1500-100ms we only have a single bucket). 

With 3 as the factor, the following buckets are generated - [10000, 3330, 1110, 370, 120, 40, 10]. This splits that 1500-100ms range into 2-3 buckets.

Hence I switched the factor to 3.

Reply:
Got it! Will switch back the factor to 5